.github/workflows/ci.yml

```diff
@@ -19,3 +19,24 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn build
       - run: yarn test
+
+  publish:
```

Comment:
not sure, with this it doesn't need to build the assets again in a separate workflow.
it takes them from the `build` job.
we could rename?

Reply:
just seems wrong for it to try and publish as part of the regular CI. As a separate workflow, it could be made to run only on pushed to the main branch.

Reply:
agree @gkatsev

Reply:
alright yes, I agree it's better to not have it cancel via `if: github.ref == 'refs/heads/main'` 
will change it